gap/ideals/ideals.gd

```diff
@@ -40,6 +53,6 @@ DeclareAttribute("MinimalIdealGeneratingSet", IsSemigroupIdeal);
 
 DeclareAttribute("SupersemigroupOfIdeal", IsSemigroupIdeal);
 
-InstallTrueMethod(IsSemigroup, IsSemigroupIdeal);
+InstallTrueMethod(IsSemigroup, IsSemigroupIdeal);  # Duplicate with ideal.gi?
```

Comment:
If it's duplicate can try just deleting it.

Reply:
Removing it seems to break things. I deleted the comment.